.eslintrc.js

```diff
@@ -80,7 +80,7 @@ const config = {
 			},
 		},
 		{
-			files: ['test/e2e/__tests__/**/*'],
+			files: ['test/e2e/__tests__/**/*', 'test/e2e/pnp/__tests__/**/*'],
```

Comment:
[suggestion] Minor refactoring (optional). Not confident, but does either one work?

```suggestion
			files: ['test/e2e/**/__tests__/**/*'],
```

or

```suggestion
			files: ['test/e2e/__tests__/**/*', 'test/e2e/*/__tests__/**/*'],
```